### ISSUE TITLE
`proxy_protocol` on `NodeBalancerConfig` is missing

### ISSUE THREAD

**Issue by muradm** [linode/linode_api4-python#189]
[Official API documentation](https://www.linode.com/docs/api/nodebalancers/#config-create) says that there is a `proxy_protocol` field on `NodeBalancerConfig`.  Trying to get/set one with `linode_api4` causes:
```
  File "python3.9/site-packages/linode_api4/objects/base.py", line 135, in __getattribute__
    return object.__getattribute__(self, name)
AttributeError: 'NodeBalancerConfig' object has no attribute 'proxy_protocol'
```

**Comment by Dorthu** [comment]
Opened a PR to fix this.  Sorry it took so long.  Should be in the next release.

**Comment by Dorthu** [comment]
Released in version 3.0.5
